spec/Section 5 -- Validation.md

```diff
@@ -1,7 +1,7 @@
 # Validation
 
 GraphQL does not just verify if a request is syntactically correct, but also
-ensures that it is unambiguous and mistake-free in the context of a given
+ensures that it is unambiguous and mistakes-free in the context of a given
```

Comment:
I think either of these wordings could work, since anything that is "something-free" is absent any "something" or any "somethings".

Reply:
Definitely prefer the original.